frontend/src/tests/chatbotStorage.test.ts

```diff
@@ -0,0 +1,87 @@
+import { loadChatbotSessions, loadChatbotLastSessionId } from '../utils/chatbotStorage';
+import { type ChatSession } from '../model/ChatSession';
+
+describe('storageUtils', () => {
+  const mockSessions: ChatSession[] = [
+    {
+      id: 'session-1',
+      messages: [],
+      createdAt: '2024-01-01T00:00:00Z',
+      isLoading: false,
+    },
+    {
+      id: 'session-2',
+      messages: [],
+      createdAt: '2024-01-02T00:00:00Z',
+      isLoading: false,
+    },
+  ];
+
+  beforeEach(() => {
+    sessionStorage.clear();
+    jest.clearAllMocks();
+  });
+
```

Comment:
Add Test for Empty Array in Storage. i.e test for when the key exists and contains [] (i.e., a valid but empty session list). It's worth asserting that the utility doesn't treat it as an error.